config/vufind/Overdrive.ini

```diff
@@ -70,6 +70,15 @@ ILSname = ""
 ; set production mode to true to begin using prod URLs and IDs
 productionMode = true
 
+; Overdrive provides two levels of API access: client and patron authentiation.
```

Comment:
Typo: authentiation.

Reply:
Also, I think the preferred capitalization is OverDrive, so you may want to do a string-replace of Overdrive --> OverDrive for consistency.

Reply:
Thanks, I fixed the capitalization here in Overdrive.ini.  I could fix the new log messages in OverdriveConnector, but then I'd probably also want to fix the *old* log messages?  (I would not touch the filenames / function names, we don't want another "fulfilment" breaking change...)

Reply:
Agreed, I'm not too worried about file or function names, but I'm trying to be consistent in things that are more user-facing like on-screen messages and documentation. It wouldn't hurt to fix the log messages too, though it's not absolutely critical. If you feel like doing it, please go ahead, but it might also be something we could tackle as a follow-up PR if we want to keep this one from getting too many unrelated changes.

Reply:
Ok I think I took care of all the non-breaking capitalization in OverdriveConnector, where it makes sense to do now since there are so many log messages already and we are adding more.